### ISSUE TITLE
Arch Linux compile error

### ISSUE THREAD

**Issue by DanRStevens** [lairworks/nas2d-core#1019]
The following compile error was reported on Arch Linux (via Discord in #ophd):
> NAS2D/Renderer/../Utility.h:58:28: error: expected unqualified-id before ‘)’ token
   58 |                 Utility<T>() = delete;

**Comment by DanRStevens** [comment]
On a hunch, I tried compiling in C++17 mode rather than C++20, and the above errors went away. There was still an error later on during the link process, so still no executable. Though considering the compile step completed successfully, it looks like the error above is related to the language standard in effect when compiling.

**Comment by DanRStevens** [comment]
I was able to confirm this issue also affects Ubuntu 22.04, which now comes with GCC 11. It seems when `g++-11` is compiling in `c++20` mode, it doesn't like the explicit template parameter when deleting `Utility` constructors.

I think I also found a fix. It would appear the code works as expected without the explicit template parameter. That would make sense, since putting it on the class itself, would kind of imply it being there for the class constructor.
